KubeArmor/nfpm.yaml

```diff
@@ -0,0 +1,32 @@
+name: "kubearmor"
+arch: "${ARCH}"
+platform: "linux"
+version: "${VERSION}"
+section: "default"
+priority: "extra"
+replaces:
+  - kubearmor
+provides:
+  - kubearmor
+maintainer: "Barun Acharya <barun.acharya@accuknox.com>"
+description: |
+  Cloud-native Runtime Security Enforcement System
+vendor: "kubearmor"
+homepage: "https://kubearmor.com"
+license: "Apache 2"
+contents:
+  - src: ./kubearmor
+    dst: /usr/local/bin/kubearmor
+  - src: ./packaging/kubearmor.conf
+    dst: /etc/sysconfig/kubearmor.conf
+    type: config
+  - src: ./packaging/kubearmor.service
+    dst: /usr/lib/systemd/system/kubearmor.service
+    type: config
+  - src: /usr/local/bin/kubearmor
+    dst: /usr/sbin/kubearmor
+    type: symlink
+  - src: ./BPF/*
+    dst: /usr/local/bin/BPF/
```

Comment:
I think this is not a good way.

why don't we create new directories (e.g., /opt/kubearmor{,/etc,/log,/BPF})
and make a symbolic link (e.g., /opt/kubearmor/kubearmor -> /usr/local/bin/kubearmor)?

Reply:
Would it be a good idea to use /usr/share/KubeArmor and we can set the working directory of systemd service to /usr/share/kubearmor?